bindata/oauth-apiserver/deploy.yaml

```diff
@@ -133,18 +135,18 @@ spec:
           periodSeconds: 5
           timeoutSeconds: 10
           successThreshold: 1
-          failureThreshold: 1
+          failureThreshold: 3
         startupProbe:
           httpGet:
             scheme: HTTPS
             port: 8443
-            path: healthz
+            path: livez?exclude=etcd
```

Comment:
blocking comment: as indicated in https://github.com/openshift/cluster-openshift-apiserver-operator/pull/588#discussion_r1755137655, it's a good idea to ensure that we have etcd working during *startup*, but it's a good idea to exclude it from ongoing checks.  This means that the startup problem should not exclude etcd and the liveness probe should exclude etcd